app/src/main/java/ru/startandroid/organizer/app/NetworkModule.kt

```diff
@@ -0,0 +1,37 @@
+package ru.startandroid.organizer.app
+
+
+
+import javax.inject.Singleton
+import dagger.Module
+import dagger.Provides
+import okhttp3.Interceptor
+import okhttp3.OkHttpClient
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+@Module
+class NetworkModule {
+
+    @Provides
+    @Singleton
+    internal fun provideNetwork(): Retrofit {
+
+        val clientBuilder = OkHttpClient.Builder()
+
+
+        val headerAuthorizationInterceptor = Interceptor { chain ->
+            var request: okhttp3.Request = chain.request()
+            val headers = request.headers().newBuilder().add("key", "405f4e5b916047bd9f4193926181511").build()
```

Comment:
Are you sure you can set key as a header, and not as a query param?

Reply:
I'm not sure, I've tried to test it, but I failed to understand how Dagger works in this project, so
I couldn't inject it into the HomeFragment...